core/src/main/scala/org/genivi/sota/core/CampaignLauncher.scala

```diff
@@ -28,17 +28,16 @@ object CampaignLauncher {
       val groupId = campGrp.group
       for {
         updateRequest <- updateService.updateRequest(ns, pkgId)
+        devices       <- deviceRegistry.fetchGroup(groupId)
```

Comment:
This end-point extracts the namespace from the token, but here we are not sending a token.